modules/ROOT/pages/6.8-release-notes.adoc

```diff
@@ -233,6 +233,11 @@ waiting for confirmation on these new feature tickets.
 
 === Pressing Enter when focusing on the `details` component for an Accordion deletes its structure.
 // #TINY-10177
+Previously, clicking on the expand arrow of a `details` element would result in the editor selection to be external of the `summary` element when the host browser was Safari.
+
+As a consequence, the `isInSummary` predicate could not detect the selection at the beginning of the `details` element and would fail to toggle the Accordion component on **Enter** key press.
```

Comment:
```suggestion
As a consequence, the plugin failed to detect if the selection was at the beginning of the `details` element, which resulted in a unsuccessful attempt to toggle the Accordion component when the **Enter** key is pressed
```

Reply:
@vpyshnenko see above suggestion for re-write

Reply:
That's good one.